openproblems/tasks/regulatory_effect_prediction/methods/beta.py

```diff
@@ -226,6 +226,6 @@ def _atac_genes_score(adata, top_genes=500, threshold=1):
     code_url="http://cistrome.org/BETA",
     image="openproblems-python-extras",
 )
-def beta(adata, n_top_genes=500, threshold=1):
-    adata = _atac_genes_score(adata, top_genes=n_top_genes, threshold=threshold)
+def beta(adata, test=False, top_genes=500, threshold=1):
+    adata = _atac_genes_score(adata, top_genes=top_genes, threshold=threshold)
```

Comment:
Don't know anything about this method, but would `top_genes` not also be a test parameter?